tslib/src/functions-throttled/sync-results-init/sync-results-init.service.ts

```diff
@@ -57,7 +57,7 @@ export class SyncResultsInitService {
     if (archive === null || archive === undefined || archive === '') {
       throw new Error(`CheckCode ${check.checkCode} has an invalid archive`)
     }
-    const payloadString = this.compressionService.decompress(archive)
+    const payloadString = this.compressionService.decompressFromUTF16(archive)
```

Comment:
Surely this is now base64 encoded (and compressed)?

Reply:
yes, see latest commits.
that was part of a rename refactor in bulk